docs/docs/metrics.md

```diff
@@ -117,37 +94,31 @@ Please note that prior to <span class="label label-default">v0.15</span> these w
+For more information, consult [the code](https://github.com/dropwizard/metrics/blob/796663609f310888240cc8afb58f75396f8391d2/metrics-jetty9/src/main/java/io/dropwizard/metrics/jetty9/InstrumentedHandler.java#L41-L42).
 
 ### JVM
 
-`jvm.threads.count` (meter) - the total number of threads. If this is above >500, this
-is generally a bad sign.
+`jvm.threads.count` (meter) - The total number of threads. This number should be below 500.
 
-`jvm.memory.total.used` (meter) - the total number of bytes used by the Marathon JVM.
+`jvm.memory.total.used` (meter) - The total number of bytes used by the Marathon JVM.
 
 ## Instrumented method calls
 
 These metrics are created automatically by instrumenting certain classes in our code base.
- This feature can be disabled
-with `--disable_metrics`. It will not disable all metrics but only this code instrumentation. 
-Generally, these timers can be very valuable in diagnosing
- unforeseen problems but require detailed insight into the inner workings of Marathon.
- They might also degrade performance noticeably.
+
+You can disable these instrumented metrics with `--disable_metrics`. This flag will only disable this code instrumentation, not all metrics.
+
+These timers can be very valuable in diagnosing problems, but they require detailed knowledge of the inner workings of Marathon. They can also degrade performance noticeably.
 
 Since these metric names directly correspond to class and method names in our code base,
 expect the names of these metrics to change if the affected code changes.
 
-## Potential issues
+## Potential pitfalls
 
 ### Derived metrics (mean, p99, ...)
 
-Our metrics library calculates derived metrics like "mean" and "p99". Unfortunately, if reported to statsd, they
-do not only relate to the reporting interval but the whole live time of the app with some exponential weighting
-algorithm. So try to build your dashboard around "counts" rather than "rates" where possible.
+Our metrics library calculates derived metrics like "mean" and "p99." However, Marathon provides these metrics for the entire live of of the app and applies an exponential weighting algorithm as a heuristic. For more precise metrics, build your dashboard around "counts" rather than "rates" where possible.
 
-### Statsd, derived statistics and metric names
+### Statsd, derived statistics, and metric names
 
-Statsd typically creates derived statistics (mean, P99) from what is reported. This might interact in a weird
-fashion with the derived statistics that our codahale metrics package reports.
+Statsd typically creates derived statistics (mean, p99) from mean values Marathon reports. Our codahale metrics package also reports derived statistics, which may differ from those created by statsd. Be sure you know where you are reporting and computing mean values.
```

Comment:
What role to metric names play here? Is the trouble that there may be different statistics with the same metrics name?